pkg/install/v1alpha1/volume_stats.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2018 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+const volumeStatsYaml = `
+---
+apiVersion: openebs.io/v1alpha1
+kind: CASTemplate
+metadata:
+  name: cas-volume-stats-default
+spec:
+  taskNamespace: {{env "OPENEBS_NAMESPACE"}}
+  run:
+    tasks:
+      - cas-volume-stats-default
+  output: cas-volume-stats-output-default
+---
+apiVersion: openebs.io/v1alpha1
+kind: RunTask
+metadata: 
+  name: cas-volume-stats-default
+spec:
+  meta: |
+    id: readvolumesvc
+    runNamespace: {{ .Volume.runNamespace }}
+    apiVersion: v1
+    kind: Service
+    action: list
```

Comment:
This needs to be read

Reply:
since we are listing the service based on labels we should not change it to list as we are not having service name instead we are selecting it based on the volume name.

Reply:
ok